src/library_dylink.js

```diff
@@ -572,7 +572,7 @@ var LibraryDylink = {
         return resolved;
       }
 
-      // TODO kill ↓↓↓ (except "symbols local to this module", it will likely be
```

Comment:
Maybe we should revert this part of the fix?  Along with the change to src/library_glfw.js?  It seems unnecessary in the face of the other, actual fix?